### ISSUE TITLE
[Package] upgrade to the latest AppInsights SDK (live metrics)

### ISSUE THREAD

**Issue by ferantivero** [mspnp/microservices-reference-implementation#134]
upgrade the to the latest AppInsights-nodejs version since it is addressing some connections problems when using live metrics: https://github.com/microsoft/ApplicationInsights-node.js/issues/510

**Comment by ferantivero** [comment]
please let's consider also updagrading the rest of the services too.

**Comment by ferantivero** [comment]
please let me cc/ @doodlemania2 to see if we can baseline this elsewhere

**Comment by doodlemania2** [comment]
I agree - please create a UserStory and Task for these @ferantivero - keeping our packages up to date is imperative. I'd even suggest we halt further development of shiny new things until we get that automated. cc: @ckittel
